profile/src/v100/address/non_fungible_local_id.rs

```diff
@@ -0,0 +1,182 @@
+use crate::prelude::*;
+
+use radix_engine_common::data::scrypto::model::NonFungibleLocalId as NativeNonFungibleLocalId;
+
+#[derive(Clone, Debug, Hash, PartialEq, Eq, uniffi::Enum)]
+pub enum NonFungibleLocalId {
```

Comment:
same here, would be nice with a newtype of `radix_engine_common::data::scrypto::model::NonFungibleLocalId` + own Serde + UniFFI.